ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Runner.scala

```diff
@@ -85,15 +88,14 @@ class Runner[Extra](name: String, constructor: LedgerFactory[Extra]) {
   private def startIndexerServer(
       config: Config[Extra],
       readService: ReadService,
-  )(implicit executionContext: ExecutionContext): Resource[Unit] =
+  )(implicit executionContext: ExecutionContext, ctx: LoggingContext): Resource[Unit] =
```

Comment:
You missed a few. Try `ag ' ctx: LoggingContext'`.

Reply:
Ah, yes, thanks.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/4097/commits/f6f0e94ba086d6dee3ad329ba3ba79d09b8d4026